### PR TITLE
Consolify snapshot documentation

### PR DESCRIPTION
**Pull request by abeyad**
This commit fixes the snapshot documentation to be in conformity
with the CONSOLE format.

### CODE REVIEW

File: docs/reference/modules/snapshots.asciidoc [hunk 1]
```diff
@@ -24,7 +24,7 @@ Elasticsearch. The repository settings are repository-type specific. See below f
 
 [source,js]
 -----------------------------------
-PUT /_snapshot/my_backup
+PUT _snapshot/my_backup
```
Comment (nik9000) [inline docs/reference/modules/snapshots.asciidoc]: The leading `/` is fine.

Comment (nik9000) [inline docs/reference/modules/snapshots.asciidoc]: No need to remove it.

File: docs/reference/modules/snapshots.asciidoc [hunk 2]
```diff
@@ -24,7 +24,7 @@ Elasticsearch. The repository settings are repository-type specific. See below f
...
 {
   "type": "fs",
   "settings": {
```
Comment (nik9000) [inline docs/reference/modules/snapshots.asciidoc]: Maybe this should have an example instead of `...`?

Comment (nik9000) [inline docs/reference/modules/snapshots.asciidoc]: And this can be `// CONSOLE` too.

File: docs/reference/modules/snapshots.asciidoc [hunk 3]
```diff
@@ -62,23 +62,26 @@ can be obtained using the following command:
...
+// CONSOLE
 
 or
 
 [source,js]
 -----------------------------------
-GET /_snapshot/_all
+GET _snapshot/_all
 -----------------------------------
+// CONSOLE
```
Comment (nik9000) [inline docs/reference/modules/snapshots.asciidoc]: Do these pass? Like, I expect most of them to 404 unless you do something to make sure the repository exists.

File: docs/reference/modules/snapshots.asciidoc (hunk 2 above)
Comment (nik9000) [inline docs/reference/modules/snapshots.asciidoc]: You could do `// CONSOLE` and `// TESTESTUP` and that way the repository is created for every snippet.

**Review by nik9000 — CHANGES_REQUESTED** [review]

**Comment by abeyad** [comment]
@nik9000 I pushed https://github.com/elastic/elasticsearch/pull/23189/commits/d14a71639e8de8503d19eb9b9a783c743ac41e80 which fixes the snapshot doc so they work with the docs tests.  I also addressed your feedback and changed the snippets to have a leading `/`.

**Review by nik9000 — APPROVED** [review]

**Pull request merged**